report.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+echo "Enter cluster name (default my-cluster)"
+read cluster
+if [ -z $cluster ]; then
+	cluster="my-cluster"
+fi
+
+get_yamls() {
+	mkdir -p ./reports/"$1"s
+	resources=`oc get "$1"s  -l strimzi.io/cluster=$cluster | tail -n +2` # remove legend
+	while read -r line; do
+		resource=`echo $line | cut -f 1 -d " "`
+		echo $resource
+		oc get "$1" $resource -o yaml >> ./reports/"$1"s/"$resource".txt
+	done <<< $resources;
+}
+
+get_yamls "deployment"
+get_yamls "statefulset"
+get_yamls "configmap"
+get_yamls "secret"
```

Comment:
We are definitely missing at least services, pod disruption budget, our own RBAC roles and their (cluster) role bindings, network policies and routes.